_default-dev-settings

```diff
@@ -6,7 +6,8 @@
 
 # Prevent cluster collisions in shared environments with a resource name prefix
 # Can safely be commented-out for local libvirt use
-export CLUSTER_PREFIX="$USER-"
+export USER=$(whoami || echo "default-user")
```

Comment:
I don't think we want a default user. That could still encourage collisions. The intent is that with `-u`, this will fail if the `USER` env var is unset.